### DISCUSSION TITLE
The repository Satyendra179 has been retired and cannot be reused.

### DISCUSSION THREAD

**Discussion by Satyendra179** [community/community#44332] [New to GitHub]
I am not able to set my github account what I do I am seeing warning like "The repository Satyendra179 has been retired and cannot be reused."

**Comment by SilasRodrigues19** [comment]
Hey @Satyendra179 

There is a resolved discussion on this subject, maybe it can help you understand.

#23480

**Reply by queenofcorgis** [reply]
Thanks @SilasRodrigues19 for linking the discussion - if this helped you @Satyendra179 don't forget to click the Mark as answer button 😄
